internal/service/redshift/cluster.go

```diff
@@ -363,7 +363,9 @@ func ResourceCluster() *schema.Resource {
 				ForceNew: true,
 			},
 			"snapshot_copy": {
-				Type:     schema.TypeList,
+				Type: schema.TypeList,
+				Deprecated: "Use the aws_redshift_snapshot_copy resource instead. " +
+					"This argument will be removed in a future major version.",
```

Comment:
Should this now also be `Computed: true`?

Reply:
Yes, thanks!

Reply:
Thanks for pointing this out - I've made the argument optional/computed and removed all the bits about `ignore_changes` from the docs and acceptance tests.